github/filecoin-project.yml

```diff
@@ -4589,165 +3907,140 @@ teams:
         - jennijuju
         - ZenGround0
       member:
-        - aarshkshah1992
         - Kubuxu
         - rjan90
         - rvagg
         - TippyFlitsUK
   advocates:
     members:
       member:
-        - hammertoe
-        - lksbrssr
         - longfeiWan9
-        - mishmosh
         - trruckerfling
   alumni:
+    {}
+  Alumni:
     members:
```

Comment:
@BigLep We could convert Alumni team members to external collaborators. The new members added in this PR haven't been active in 12 months. Where activity is defined as follows - https://github.com/filecoin-project/github-mgmt/blob/master/scripts/src/actions/remove-inactive-members.ts#L25